.github/workflows/checks.yml

```diff
@@ -11,6 +11,12 @@ jobs:
       - uses: actions/checkout@v2
       - name: Install uv
         uses: astral-sh/setup-uv@v6
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
+      - name: Install doc requirements
+        run: pip install -r docs/requirements.txt
```

Comment:
uv is installed in the prior command, using that would probably be better - happy to help get this working if you get stuck!